dev/preview/BUILD.yaml

```diff
@@ -30,6 +30,7 @@ scripts:
       export TF_VAR_vm_memory="${TF_VAR_vm_memory:-12Gi}"
       export TF_VAR_vm_storage_class="${TF_VAR_vm_storage_class:-longhorn-gitpod-k3s-202209251218-onereplica}"
       ./workflow/preview/deploy-harvester.sh
+      previewctl install-context
```

Comment:
```suggestion
      previewctl install-context --watch
```

Reply:
@vulkoingim I don't think we want that - if we use `--watch` then it will never exit right? It will go into a watch loop and just continue to log `time="2022-11-08 14:20:18" level=info msg="The preview envrionment hasn't changed"`